src/containers/Guide/GuideBlocks/index.tsx

```diff
@@ -1,108 +1,118 @@
 import React, {FC} from 'react';
+import {NavLink} from 'react-router-dom';
 
-import {A, DocContainer, DocImage, DocList, TableBorderGrid} from 'components';
+import {
+  A,
+  CalloutType,
+  DocCallout,
+  DocContainer,
+  DocImage,
+  DocList,
+  DocSubHeader,
+  TableBorderGrid,
+  TableVertical,
+} from 'components';
 
 import BalanceLockAndKey from './BalanceLockAndKey.png';
 import BlockDetails from './BlockDetails.png';
 import './Blocks.scss';
 
 const GuideBlocks: FC = () => {
   return (
-    <DocContainer className="GuideBlocks" title="Blocks" lastUpdated="07 Dec 2020">
+    <DocContainer className="GuideBlocks" title="Blocks" lastUpdated="17 Feb 2021">
       <p>
-        A block is a group of one or more transactions. There are often multiple transactions (usually fees) to
-        different recipients within a single block. We can inspect the structure of a block by first examining a
-        simplified overview below. In this example, Amy is sending 100 coins to her friend Brian.
+        A block is a group of one or more transactions. There are often multiple transactions (usually fees) with
+        different recipients within a single block. Let's inspect the structure of a block by first examining the
+        simplified scenario below.
       </p>
+      <DocSubHeader>Example</DocSubHeader>
+      <p>In this example, Amy sends 100 coins to Brian:</p>
 
       <TableBorderGrid
         rows={[
-          ['Coins being sent to Brian', '100'],
+          [<strong>Description</strong>, <strong>Coins</strong>],
+          ['Coins sent to Brian', '100'],
           ['Bank fees', <span className="GuideBlocks__text-light">2</span>],
           ['Validator fees', <span className="GuideBlocks__text-light">1</span>],
```

Comment:
@led76marx what do you think about using light-grey for this? I always thought it kind of odd

Reply:
Oh, yeah. It is kind of weird. But not a really big deal. I think that the main point in the orginal text is not to pay attention to Transaction fees, and I wold keep light-gray only for that reason, cause it's not a topic covered in this doc

Reply:
Got rid of the light-grey